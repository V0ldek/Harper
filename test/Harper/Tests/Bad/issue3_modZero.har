main = f 0;

f n = 42 mod n;