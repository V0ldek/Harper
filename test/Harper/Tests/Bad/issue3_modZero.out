Error: expression `n` evaluated to zero causing a division error.
During evaluation of:
 42 mod n 
Located at line 3 column 7

Execution terminated with an error: runtime error.